integrations/extensions/starter-kits/testitall/testitall.actions.json

```diff
@@ -150,9 +145,9 @@
                 "type": "integration_interaction",
                 "method": "POST",
                 "internal": {
-                  "spec_hash_id": "837b4f9e9cbcd25948117367cbee4ddcbf870a48496df7e15ad127852843499f",
-                  "match_scenario": 12,
-                  "catalog_item_id": ""
+                  "spec_hash_id": null,
```

Comment:
The `spec_hash_ids` cause the existing extensions to fail, can you recreate this Action Skills JSON with the hash IDs?

Reply:
done